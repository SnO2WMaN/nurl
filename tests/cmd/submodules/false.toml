args = [
    "https://github.com/Koihik/LuaFormatter",
    "1.3.0",
    "--submodules=false",
    "--fetcher",
    "fetchgit",
]
