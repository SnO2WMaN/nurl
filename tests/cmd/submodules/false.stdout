fetchgit {
  url = "https://github.com/Koihik/LuaFormatter";
  rev = "1.3.0";
  hash = "sha256-UrADnTFH8y1rqA2LytSK9HkIQdxOIrY4CwqfL139Oag=";
  fetchSubmodules = false;
}